### DISCUSSION TITLE
Can't make it work

### DISCUSSION THREAD

**Discussion by nuthinking** [mixpanel/mixpanel-js#277] [General]
Tried snippet and npm and can't see any data. I disabled also the adblocker 🤷‍♂️: https://community.mixpanel.com/sending-data-to-mixpanel-11/getting-data-from-javascript-5003

**Comment by nuthinking** [comment]
Despite the adblocker was disabled for the site, the "Do not track" was still enabled and prevented the tracking.

**Comment by tdumitrescu** [comment]
Yeah, the default settings respect Do Not Track. If you want to disable that on your site, you can use the config flag `ignore_dnt: true`
